sdk/python/kfp/components/_components.py

```diff
@@ -301,7 +301,7 @@ def expand_command_part(arg): #input values with original names
                     assert isinstance(func_argument, str)
                     input_name = func_argument
                     pythonic_input_name = input_name_to_pythonic[input_name]
-                    argument_is_present = pythonic_input_name in pythonic_input_argument_values
+                    argument_is_present = pythonic_input_argument_values[pythonic_input_name] is not None
                     return str(argument_is_present)
```

Comment:
Maybe change the format of the returning variable to bool instead of string?

Reply:
I've been thinking about this too.
This is not as trivial since right now the value can come from outside and might be a string. But I'm thinking about a best way to handle this.
I might try it when I'm adding support for default values.

Reply:
SG